src/protections/WordList.ts

```diff
@@ -193,12 +193,16 @@ export class WordListProtection
 
       const match = this.badWords.exec(message);
       if (match) {
-        const reason = `bad word: ${match[0]}`;
+        const reason = `Said a bad word. Moderators, consult the management room for more information.`;
         await this.userConsequences.consequenceForUserInRoom(
           roomID,
           event.sender,
           reason
         );
+        await this.draupnir.client.sendMessage(this.draupnir.managementRoomID, {
+          msgtype: "m.notice",
+          body: `Banned ${event.sender} in ${roomID} for saying '${match[0]}'.`,
+        });
```

Comment:
Could you catch this if it errors?

One way could be like this:

```suggestion
        await this.draupnir.client.sendMessage(this.draupnir.managementRoomID, {
          msgtype: "m.notice",
          body: `Banned ${event.sender} in ${roomID} for saying '${match[0]}'.`,
        }).then((_) => Ok(undefined), resultifyBotSDKRequestError);
```

You could also do something like this if you're feeling extra diligent but i don't think there's a need really

```suggestion
        const messageResult = await this.draupnir.client.sendMessage(this.draupnir.managementRoomID, {
          msgtype: "m.notice",
          body: `Banned ${event.sender} in ${roomID} for saying '${match[0]}'.`,
        }).then((_) => Ok(undefined), resultifyBotSDKRequestError);
        if (isError(messageResult) {
          log.error("Failed to send a message to the management room", messageResult.error)
        }
```

Reply:
Might implement the latter but modify it to include the banned word in the log too, just so that it doesn't have the possibility of simply being lost.

Reply:
Okie, that's a great idea ^^